### ISSUE TITLE
Electron target version in app/package.json doesn't match installed Electron version

### ISSUE THREAD

**Issue by aguynamedben** [chentsulin/electron-react-boilerplate#1159]
I think I found a minor issue where the latest version of Electron, version 1.7.5, which is installed with `yarn` based on package.json's specifications, but then there is a mismatch between the Electron version and the `--target` argument passed in app/package.json's use of `npm rebuild`. This causes a runtime error related to `NODE_MODULE_VERSION` when you try to use native modules:

```
Uncaught (in promise) Error: Error: The module '/Users/ben/code/tesla/app/node_modules/leveldown/build/Release/leveldown.node'
was compiled against a different Node.js version using
NODE_MODULE_VERSION 53. This version of Node.js requires
NODE_MODULE_VERSION 54. Please try re-compiling or re-installing
the module (for instance, using `npm rebuild` or `npm install`).: unable to import leveldown
```

## How to Reproduce

1. Clone the project
1. Run `yarn` to install dependencies
1. Note that `yarn list | grep electron` will show electron@1.7.5 installed. This differs from the `yarn.lock` on the project's GitHub. Note that `devDependencies` in the project's package.json (top-level) specifies `"electron": "^1.6.10",`
1. `cd app`
1. `yarn add pouchdb` (we add PouchDB in app/package.json because the LevelDB native adapter will also be installed, see https://github.com/chentsulin/electron-react-boilerplate/wiki/Module-Structure----Two-package.json-Structure)
1. `cd ..`
1. `yarn`
1. Add some code in app/index.js that uses PouchDB:
```javascript
// pouchdb
import PouchDB from 'pouchdb';
var localDB = new PouchDB('foo');
localDB.info().then(function (info) {
  console.log(info);
});
```
1. `yarn dev` to run the app
1. Inspect the dev tools JavaScript console. You will see the following error:
<img width="858" alt="developer_tools_-_file____users_ben_code_tesla_app_app_html" src="https://user-images.githubusercontent.com/107841/28890793-22b5f54a-777d-11e7-92a4-366d49a906df.png">
From what I understand, this error occurs when a native module (LevelDB in this case, a dependency of PouchDB) is built targeting a different version of Node.js than Electron is using.

## How to Fix

1. In app/package.json, in the "scripts" section, change the --target argument from 1.6.6 to 1.7.5:
```json
    "postinstall": "npm rebuild --runtime=electron --target=1.7.5 --disturl=https://atom.io/download/atom-shell --build-from-source"
```
1. `cd app`
1. `yarn`
1. `cd ..`
1. `yarn`
1. `yarn dev`

## Pull Request

Should I submit a pull request with this small change?

**Comment by amilajack** [comment]
Sure. Make sure to make the PR against the `v0.13.0` branch

**Comment by aguynamedben** [comment]
It looks like 40a05d620d21023e5317eab1524028d7029e3908 might address it. I'll help QA the `v0.13.0` branch some today. Thanks for your work on this project.

**Comment by pizzafroide** [comment]
40a05d6 chooses version `1.7.0` but, today, it is version `1.7.6` that will be installed (and, ABI changes between versions `1.7.5` and `1.7.6`). So this kind of change is not really durable. Why not dropping the `--version` option from `electron-rebuild` call? `electron-rebuild` is supposed to locate `electron` installation and deduce its version number anyway.

**Comment by aguynamedben** [comment]
@pizzafroide I agree, and today it's `1.8.1` and I had to change it to that.

It looks like @amilajack has fixed this in the v0.13.0 branch: https://github.com/chentsulin/electron-react-boilerplate/blob/v0.13.0/app/package.json.

👏  @amilajack for mayor!!!

**Comment by aguynamedben** [comment]
(exact commit that makes the change: https://github.com/chentsulin/electron-react-boilerplate/commit/83246d7e2971950c175316be498942907b863ebf#diff-ac6ee8c82c5d20d9bb9c4d6ac70161d4)
